Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM talis/ubuntu:1404-latest
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install -y --force-yes \
+        curl \
+        git \
+        apt-transport-https \
+        nodejs=8.9.4-1nodesource1 \
+        python \
+        build-essential \
+        envconsul \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+
+# Tidy up
+RUN apt-get -y autoremove && apt-get clean && apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
This should run together we the above RUN. 

Creating a layer that removes things only hides those files for the next layer, but they are still present in the Docker image (or at least unless the image is [`--squash`ed](https://docs.docker.com/engine/reference/commandline/image_build/)).

Reply:
I'll change that, in fairness, I was copying what we have in many of our other Dockerfiles. We might need to retrospectively correct those as well.